### DISCUSSION TITLE
Improve Notification to better handle shortcode

### DISCUSSION THREAD

**Discussion by rizaardiyanto1412** [publishpress/PublishPress-Planner#1374] [Feature Requests]
Currently, our Notificaiton can't handle shortcode that coming from other plugins. 

It will be great if we can render that as well

perhaps notification body content can be piped thru wp’s do_shortcode()

Original request: https://wordpress.org/support/topic/shortcodes-in-content-3/

**Comment by airdrummer** [comment]
this would be of great help to me, as i use [wpmember] shortcodes to restrict access to some content.
